src/components/DiffView/styles.module.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../scss/variables';
+
+.diff {
+  border-bottom-left-radius: $border-radius;
+  border-bottom-right-radius: $border-radius;
+  border-collapse: separate !important;
```

Comment:
Can you use specificity instead of `!important`?

Reply:
I could add a comment at least, because it overrides the `react-diff-view` style. But maybe I could be more specific.